webui/react/src/components/TableActionBar.tsx

```diff
@@ -154,39 +145,52 @@ const TableActionBar: React.FC<Props> = ({
     useModal(ExperimentTensorBoardModal);
   const isMobile = useMobile();
   const { openToast } = useToast();
-  const experimentIds = useMemo(
-    () => Array.from(selectedExperimentIds.keys()),
-    [selectedExperimentIds],
-  );
+
+  const [experiments, setExperiments] = useState<Loadable<BulkExperimentItem>[]>([]);
+
+  const fetchExperimentsByIds = useCallback(async (selectedIds: number[]) => {
+    const CHUNK_SIZE = 80;
+    const requests = _.chunk(selectedIds, CHUNK_SIZE);
```

Comment:
`requests` isn't an accurate name for this variable. probably `chunkedExperimentIdList` or something?